backend/tests/integration/test_b07_p5_bayesian_timeout_runtime.py

```
@@ -0,0 +1,267 @@
+                "correlation_id": correlation_id,
+                "run_seconds": 60,
+                "continue_after_soft_timeout": True,
+            },
+            task_id=run_task_id,
+            queue="attribution",
+        )
+        run_error_class = ""
+        run_error_message = ""
+        try:
+            run_result.get(timeout=20)
+            raise AssertionError("Expected runaway Bayesian task to be hard-terminated")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)